tests/acceptance/bs-navbar-test.js

```diff
@@ -0,0 +1,13 @@
+import { test } from 'qunit';
+import moduleForAcceptance from '../../tests/helpers/module-for-acceptance';
+
+// TODO: Write this
+moduleForAcceptance('Acceptance | bs navbar');
+
+test('visiting /navbars', function(assert) {
+  visit('/navbars');
+
+  andThen(function() {
+    assert.equal(currentURL(), '/navbars');
+  });
+});
```

Comment:
You can test components with sub components as well in an integration test, see https://github.com/kaliber5/ember-bootstrap/blob/master/tests/integration/components/bs-accordion-test.js#L44-L59 as an example.